tests/kafkatest/services/kafka/quorum_utils.py

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# the types of metadata quorums we support
```

Comment:
I like the idea here.  I think we could make this less verbose, though:
1. quorum.py instead of quorum_utils.py
2. zk, raft, colocated_raft instead of the current names.  If you use "import kafkatest.services.kafka.quorum_utils" then you can refer to these things as quorum.zk, quorum.raft, quorum.colocated_raft which feels natural.